build.proj

```diff
@@ -3,8 +3,11 @@
   <PropertyGroup>
     <!-- Capture OSGroup passed to command line for setting default FilterToOSGroup value below -->
     <_OriginalOSGroup>$(OSGroup)</_OriginalOSGroup>
-    <TestOsTarget Condition="'$(OSGroup)'!=''">$(OSGroup)</TestOsTarget>
-    <TestOsTarget Condition="'$(OSGroup)'==''">$(OSEnvironment)</TestOsTarget>
+  </PropertyGroup>
+  <PropertyGroup>
+    <InputOSGroup>$(OSGroup)</InputOSGroup>
+    <InputOSGroup Condition="'$(InputOSGroup)'==''">$(FilterToOSGroup)</InputOSGroup>
+    <InputOSGroup Condition="'$(InputOSGroup)'==''">$(OS)</InputOSGroup>
```

Comment:
there is no build.props but there is a dir.props which sets OSEnviroment. In general we have tried to use OSEnviroment through all our projects and targets because we define it from OS in dir.props. In general we should use OSEnviroment, unless OS is the only way you can do it like in your default configurations that happen before the import of dir.props.